module-4/backend/week_two.md

```diff
@@ -1,22 +1,59 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+syntax for interpolation
+ES5 - combo of +'s, empty strings for spaces, and calling variables with their properties
+ES6 - backticks and ${}
+
+2. What's the difference between asynchronous and synchronous JavaScript?
+JS is synchronous - it's not a multi-threaded language
```

Comment:
Ha I wouldn't say it is a fraud. Async here is that the program can continue running and it isn't stopped while waiting for a function to finish.